logistics/venue.md

```diff
@@ -0,0 +1,99 @@
+# Venue Logistics
+
+RustBridge is an interactive event and your space should allow for that. That
+means you should have space for people to walk around and people to comfortably
+sit in groups.
+
+Learning works better when having a little space and it's not too noisy.
+
+Don't stress out about venue details _too_ much, people are great at working
+around flaws, as long as you and the attendees are aware. It's the learning 
+experience that counts.
+
+## A venue is a relationship
+
+Be aware that venues are run by people and you want to be on good terms with
+them. If you want to run multiple events and you are happy with a venue,
+you totally want to leave a lasting impression. This will make searching
+for the next time easier.
+
+If you want to run regular workshops, make sure you build a couple of contacts.
+
+## Qualities of a good venue
+
+RustBridge is a flexible program, so you are also flexible in a venue.
+
+A venue for RustBridge should fulfill the following requirements:
+* It's free
+* Space for at least 20 people to comfortably sit around tables
+* Open a little before and after the workshop
+* A bare room with rearrangable furniture is preferred
+  - avoid row seating, but take it if you have nothing else
+* Check accessibility, especially if access to the venue is step-free
+* Prefer places that have a screen or a projector to present things
+* Long events mean that people have to make pauses and step out, see
+  that there's a space to retreat
+* Have power sockets at the tables
+* Good the internet connection, Rust installation is bandwidth-intensive
```

Comment:
Also worth asking people to install stuff before arriving (but not making it a hard requirement)

Reply:
```patch
- Good the internet connection
+ Check the internet connection
```